src/test/java/marquez/api/resources/DatasetResourceTest.java

```diff
@@ -0,0 +1,76 @@
+package marquez.api.resources;
+
+import static javax.ws.rs.core.Response.Status.OK;
+import static marquez.common.models.Description.NO_DESCRIPTION;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import marquez.api.models.DatasetResponse;
+import marquez.api.models.DatasetsResponse;
+import marquez.common.models.DatasetUrn;
+import marquez.common.models.Namespace;
+import marquez.core.exceptions.UnexpectedException;
+import marquez.core.services.NamespaceService;
+import marquez.service.DatasetService;
+import marquez.service.models.Dataset;
+import org.junit.Test;
+
+public class DatasetResourceTest {
+  private static final DatasetUrn DATASET_URN = DatasetUrn.of("urn:a:b.c");
+  private static final Instant CREATED_AT = Instant.now();
+  private static final Namespace NAMESPACE = Namespace.of("test");
+  private static final Integer LIMIT = 100;
+  private static final Integer OFFSET = 0;
```

Comment:
Although these parameters don't come from the user, it still might be worth adding a couple negative tests for invalid values of limit and offset, at least to guarantee there'll be test noise if we inadvertently start to ignore/misuse those values at the DAO layer.